website/docs/patterns/description-list-patterns/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,123 @@
+## Typography
+
+List items have two visual style options: standard or callout. Both share typography styling for the _title_ but the typography for the _value_ is adjusted.
```

Comment:
I know we don't typically have a heading followed by a heading, but I can't come up with anything helpful that is generic to both title and value. 
```suggestion
```